polynote-kernel/src/main/scala/polynote/config/PolynoteConfig.scala

```diff
@@ -190,7 +191,8 @@ object SparkConfig {
     properties.get("sparkSubmitArgs"),
     None,
     None,
-    None
+    None,
+    properties.get("jarsAsUrls").map(_.toLowerCase.trim != "false")
```

Comment:
Did you intend for this to be configurable as a "magic" spark configuration? That's what this does.

Reply:
No to be honest i just looked at how spark submit args worked and emulated that. I see PolynoteConfigSpec still passes even without this. So i'll remove it